code/modules/assembly/holder.dm

```diff
@@ -21,6 +21,7 @@
 	attach(A2,user)
 	name = "[A.name]-[A2.name] assembly"
 	update_icon()
+	feedback_add_details("assembly_made","[name]")

```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code